travel_platform/test_full_landing/travel_platform.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
```

Comment:
As I wrote - the travel_platform.html is full page instead of main.html, which is a block for inserting into base.html